### PR TITLE
Feature/reroute view 2

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

**Comment by CloudNiner** [comment]
See commit log for details

File: src/nih_wayfinding/app/scripts/views/navbar/navbar-controller.js [hunk 1]
```diff
@@ -54,6 +55,10 @@
             $state.go(stateName, stateParams);
         }
 
+        function onButtonClicked() {
+            $rootScope.$broadcast(NavbarConfig.events.buttonclicked);
```
Comment (sharph) [inline src/nih_wayfinding/app/scripts/views/navbar/navbar-controller.js:59]: As somewhat of an outsider to this project at the moment, I'm not sure which button this refers to and the naming might be a little generic.

Comment (sharph) [inline src/nih_wayfinding/app/scripts/views/navbar/navbar-controller.js:59]: Oh I see, the `rightButton`. Still might be worth renaming the event.

**Comment by sharph** [comment]
+1

File: src/nih_wayfinding/app/scripts/views/navbar/navbar-controller.js (hunk 1 above)
Comment (CloudNiner) [inline src/nih_wayfinding/app/scripts/views/navbar/navbar-controller.js:59]: yeah, definitely vague. Will do.

**Pull request closed without merging**